engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -56,7 +56,10 @@ abstract class CromwellRootActor(gracefulShutdown: Boolean, abortJobsOnTerminate
   import CromwellRootActor._
   
   // Make sure the filesystems are initialized at startup
-  val _ = CromwellFileSystems.instance
+  locally(CromwellFileSystems.instance)
+  // Fail immediately if call caching configuration is invalid
+  // TODO: Find a better way to do this
+  locally(cromwell.engine.workflow.lifecycle.execution.callcaching.FileBatchSize)
```

Comment:
I'm curious what made this single option required here?

Reply:
I wanted Cromwell to not even start if this configuration is invalid. I imagine we do this differently for other configuration but this is the simplest place I could find where to fail so that we stop